minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-resources/pom.xml

```diff
@@ -25,6 +25,49 @@
     <description>holds common resources used to build installers</description>
     <build>
         <plugins>
+        	<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-antrun-plugin</artifactId>
+				<dependencies>
+					<dependency>
+						<groupId>ant-contrib</groupId>
+						<artifactId>ant-contrib</artifactId>
+						<version>20020829</version>
+					</dependency>
+				</dependencies>
+				<executions>
+					<execution>
+						<id>download-commons-daemon</id>
+						<phase>compile</phase>
+						<goals>
+							<goal>run</goal>
+						</goals>
+						<configuration>
+							<tasks>
+								<taskdef resource="net/sf/antcontrib/antcontrib.properties" />
+								<if>
+									<available file="${basedir}/src/main/resources/bin/minifiw.exe" />
+									<then>
+										<echo message="Skipping download of commons-daemon." />
+									</then>
+									<else>
+										<get
```

Comment:
What do you think about adding a checksum/hash verification of the download to verify our build is okay?

The Ant docs have a macro that accomplishes something like this: https://ant.apache.org/manual/Tasks/get.html

Reply:
I think that's a good idea.